cmake/develop_dependencies.cmake

```diff
@@ -7,29 +7,22 @@ include( FetchContent )
 
 FetchContent_Declare( ENDFtk
     GIT_REPOSITORY  https://github.com/njoy/ENDFtk
-    GIT_TAG         origin/build/fetchcontent
+    GIT_TAG         origin/master
     GIT_SHALLOW     TRUE
     )
 
 FetchContent_Declare( catch-adapter
     GIT_REPOSITORY  https://github.com/njoy/catch-adapter
-    GIT_TAG         origin/build/fetchcontent
+    GIT_TAG         origin/master
     GIT_SHALLOW     TRUE
     )
 
 FetchContent_Declare( range-v3-adapter
     GIT_REPOSITORY  https://github.com/njoy/range-v3-adapter
-    GIT_TAG         origin/build/fetchcontent
+    GIT_TAG         origin/master
     GIT_SHALLOW     TRUE
     )
 
-FetchContent_Declare( lipservice 
```

Comment:
So I can see that @ameliajo uses `lipservice` in the top-level test to convert input parameters to JSON. While this works, the test could just create a JSON object directly and eliminate the need for the lipservice dependency. 

This is project belongs to @ameliajo I'll let her decide the way she wants to do this.

Reply:
If it doesn't hurt, I'd like to keep it in there. It makes it really easy to add in new tests and make sure you're comparing against the right values (I keep the inputs-to-compare-against in the top level test directory as well)